src/js/ripple/base.js

```diff
@@ -21,7 +22,13 @@ extend(Base, {
   VER_ACCOUNT_PUBLIC: 35,
   VER_ACCOUNT_PRIVATE: 34,
   VER_FAMILY_GENERATOR: 41,
-  VER_FAMILY_SEED: 33
+  VER_FAMILY_SEED: 33,
+
+  // 6 byte version to give `sEd` prefix to 16 byte seed encodings.
+  // VER_ED25519_SEED: [ 73, 141, 118, 70, 140, 214]
+
+  // 3 byte version to give `sEd` prefix to 16 byte seed encodings.
+  VER_ED25519_SEED: [1, 225, 75]
```

Comment:
This needs stability tests for say all (2 ** 128) + n, for values of n between 0, 255?